node_helper.js

```diff
@@ -5,6 +5,7 @@ const fetch = (...args) => import('node-fetch').then(({default: fetch}) => fetch
 const moment = require("moment-timezone")
 const ICAL = require("ical.js")
 const IcalExpander = require('ical-expander')
+const base64 = require('base-64');
```

Comment:
For me it worked without adding it. Looking at https://www.npmjs.com/package/base64 it states that " Update: this module may no longer be necessary as nodejs includes its own base64 encoding/decoding functions."

Reply:
Good catch - I guess I should setup at least some CI to never miss something like this.

Reply:
It works because `base-64` is in the dependency tree of MagicMirror². I've created an PR #142 to adapt the `package.json` and an issue #144 to replace that dependency. @TheGreyDiamond: Since you've added the new (valuable) feature :+1:. Do you perhaps feel like adapting it?

Reply:
Sure, I'll look into it this afternoon!

Reply:
Cool :rocket: Let's continue the conversation in #144, if necessary.